apps/common/enums.py

```diff
@@ -7,3 +7,16 @@ class JSENDStatus(str, Enum):
     SUCCESS = "success"
     FAIL = "fail"
     ERROR = "error"
+
+
+class MultipartSubtypeEnum(Enum):
```

Comment:
from this enum we use only `MultipartSubtypeEnum.mixed`. Maybe we don't need to create enum, and just use this string value? or in future we need it?

Reply:
Of course I don't know everything about fastapi-mail functionality, but this code and that commented above was taken from oficial github repo concerning this version of fastapi-mail, and there is a possibility for involving this data in some configuring message sending